screens/onboarding/signup.js

```diff
@@ -145,6 +147,7 @@ export default function SignUpScreen({navigation, route}) {
   }
 
   function isValid() {
+    const name = `${firstName.trim()} ${lastName.trim()}`;
     return (
       name.trim() !== '' &&
```

Comment:
If we're only checking the concatenated first/last name, then this does NOT catch if the user leaves one of the fields blank. If first AND last name are required, we should check each field separately, explicitly